packages/components/src/calendar-react-day-picker/date-range-calendar.stories.tsx

```diff
@@ -0,0 +1,343 @@
+import { fn } from '@storybook/test';
+import { fr, ja, es, ko, pt, ar, it } from 'date-fns/locale';
+import { useState } from 'react';
+import { DateRangeCalendar } from './index';
+import type { Meta, StoryObj } from '@storybook/react';
+
+const meta: Meta< typeof DateRangeCalendar > = {
+	title: 'Components/DateRangeCalendar',
+	component: DateRangeCalendar,
+	tags: [ 'autodocs' ],
+	parameters: {
+		controls: { expanded: true },
+	},
+	argTypes: {
+		locale: {
+			options: [ 'ar', 'es', 'fr', 'ja', 'ko', 'pt', 'it' ],
+			mapping: {
+				ar: ar,
+				es: es,
+				fr: fr,
+				ja: ja,
+				ko: ko,
+				pt: pt,
+				it: it,
+			},
+			control: {
+				type: 'select',
+			},
+		},
+		labels: {
+			control: false,
+		},
+		defaultSelected: { control: { type: 'date' } },
+		selected: { control: { type: 'date' } },
```

Comment:
A nitpick, but we could use `selectedFrom` and `selectedTo` for the date range in the storybook?

Reply:
We prefer to keep Storybook controls as close as possible to the component's APIs. Also, I'll need to investigate why I had to add control types manually, since I would normally expect Storybook to extract them from the TypeScript prop types

Reply:
Yup, the simplest way is to use the standard object control (hoping that it can support `Date` objects), although that would be poor UI.

Alternatively, I can look into creating a custom control (which shouldn't be too difficult). I'll timebox an effort on the latter, and otherwise fallback on the former

Reply:
I ended up disabling the `selected` and `defaultSelected` controls for the date range picker because:

- it's not possible to create custom controls;
- splitting them into `selected.from` and `selected.to` leads to type errors and other complications that I'd prefer to avoid, in order to keep the example code as clean as possible;
- leaving them with the standard object control is not particularly ergonomic, since the only format that the user would be able to input for a date is the ISO string.

We can look into a more refined approach a later point if necessary.